examples/job/get-file-from-job.py

```diff
@@ -0,0 +1,282 @@
+from utils import wait_for_jobs_to_finish, get_property_by_subworkow_and_unit_indicies, dataframe_to_html, ensure_packages_are_installed
+ensure_packages_are_installed()
+
+# Relevant functions from the API client
+from exabyte_api_client.endpoints.jobs import JobEndpoints
+from exabyte_api_client.endpoints.projects import ProjectEndpoints
+from exabyte_api_client.endpoints.materials import MaterialEndpoints
+from exabyte_api_client.endpoints.bank_workflows import BankWorkflowEndpoints
+from exabyte_api_client.endpoints.raw_properties import RawPropertiesEndpoints
+
+
+# #### Materials
+#     
+# - **MATERIALS_PROJECT_IDS**: a list of material IDs to be imported from materials project
+# - **TAGS**: a list of [tags](https://docs.exabyte.io/entities-general/data/#tags) to assign to imported materials
+# - **MATERIALS_SET_NAME**: the name of the materials set
+# 
+
+# In[]:
+
+
+MATERIALS_PROJECT_ID = ["mp-32"]
+MATERIALS_SET_NAME = "materials-set"
+TAGS = ["tag1", "tag2"]
+
+
+# #### Jobs
+# 
+# Parameters for the jobs to be ran for the imported materials:
+# 
+# - **JOB_NAME_PREFIX**: prefix to be used for the job name with "{JOB_NAME_PREFIX} {FORMULA}" convention (e.g.  "Job Name Prefix - SiGe")
+# - **JOBS_SET_NAME**: the name of the jobs set
+
+# In[]:
+
+
+JOB_NAME_PREFIX = "Ge_Test"
+JOBS_SET_NAME = "jobs-set"
+
+
+# #### Workflow
+# 
+# This example is based on [this](https://platform.exabyte.io/analytics/workflows/SnGspffD7nPsBHJaQ) bank workflow which is later copied to the account workflows collection.
+
+# In[]:
+
+
+BANK_WORKFLOW_ID = "SnGspffD7nPsBHJaQ"
+
+
+# #### Compute
+# 
+# Setup compute parameters. See [this](https://docs.exabyte.io/infrastructure/compute-settings/ui) for more information about compute parameters.
+# 
+# - **NODES**: Number of nodes. Defaults to 1.
+# - **PPN**: Number of MPI processes per each node, Defaults to 1.
+# - **QUEUE**: The name of queue to submit the jobs into. Defaults to D.
+# - **TIME_LIMIT**: Job walltime. Defaults to "01:00:00" (one hour).
+# - **CLUSTER**: The full qualified domain name (FQDN) or alias of the cluster to submit the jobs into.
+
+# In[]:
+
+
+PPN = "1"
+QUEUE = "D"
+NODES = "1"
+TIME_LIMIT = "01:00:00"
+CLUSTER = "cluster-001"
+
+
+# ### Initialize endpoints
+
+# In[]:
+
+
+job_endpoints = JobEndpoints(*ENDPOINT_ARGS)
+project_endpoints = ProjectEndpoints(*ENDPOINT_ARGS)
+material_endpoints = MaterialEndpoints(*ENDPOINT_ARGS)
+raw_property_endpoints = RawPropertiesEndpoints(*ENDPOINT_ARGS)
+bank_workflow_endpoints = BankWorkflowEndpoints(*ENDPOINT_ARGS)
+
+
+# Next, we retrieve the owner and project IDs as they are needed by the endpoints. Account's default material is used to extract the owner ID. One can extract the owner ID from any other account's [entities](https://docs.exabyte.io/entities-general/overview/).
+
+# In[]:
+
+
+owner_id = material_endpoints.list({"isDefault": True, "owner._id": ACCOUNT_ID})[0]["owner"]["_id"]
```

Comment:
I think we should squash the create materials, Create workflow, Import materials, 
Create jobs and run job parts into a single cell, and redirect to another notebook if readers want to learn more about these